app/views/content/funding-and-support/if-youre-disabled.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "If you're disabled"
+heading: "If you're disabled"
+description: |-
+    Find out what extra support and adjustments you could be entitled to when you're training to teach if you're disabled.
+related_content:
+    Who do you want to teach?: "/train-to-be-a-teacher/who-do-you-want-to-teach"
+    Becoming a teacher with a hearing impairment: "/blog/overcoming-challenges-to-become-a-teacher-hearing-impairment"
+    Becoming an autistic teacher: "/blog/overcoming-challenges-to-become-a-teacher-autism"
+promo_content:
+    - content/funding-and-support/promos/get-adviser-support-promo
+navigation: 20.20
+navigation_description: Find out what extra support and adjustments you could be entitled to when you're training to teach if you're disabled.
+keywords:
+    - Disability
+    - Disabilities
+    - DSA
+    - Financial Support
+    - Disabled students
+    - Disabled Students’ Allowances
+---
+
+The experience and perspective of a diverse workforce is valued in school culture.
+
+If you’re disabled, have a mental health condition or educational needs, you can get support to become a teacher.
+
+## Disabled Students’ Allowance and other support
```

Comment:
Unsure where the content came from - this was added by Valentine. I added in the copy around how long it can take to get it, which comes from the gov.uk guidance. But Chris gave the go ahead for the content.